eng/pipelines/coreclr/perf.yml

```diff
@@ -41,9 +45,9 @@ extends:
       - template: /eng/pipelines/coreclr/perf-wasm-jobs.yml
         parameters:
           collectHelixLogsScript: ${{ variables._wasmCollectHelixLogsScript }}
-          # ${{ and(ne(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'Schedule')) }}:
-          #   runProfile: 'non-v8'
-          ${{ if ne(variables['System.TeamProject'], 'public') }}: # , notin(variables['Build.Reason'], 'Schedule')
+          ${{ and(ne(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'Schedule')) }}:
+            runProfile: 'non-v8'
```

Comment:
This can remain commented out, because it doesn't quite work, and we haven't tried to make it work either.